.storybook/main.js

```diff
@@ -3,6 +3,11 @@ const json = require('@rollup/plugin-json');
 const { importMetaAssets } = require('@web/rollup-plugin-import-meta-assets');
 const { storybookRollupPlugin } = require('../src/stories/lib/markdown.cjs');
 
+let createIconAssetsPlugin;
+import('../rollup/rollup-plugin-icon.js').then(module => {
+  createIconAssetsPlugin = module.createIconAssetsPlugin;
+});
```

Comment:
`rollup/rollup-plugin-icon.js` is an ES module file whereas `.storybook/main.js` is a CommonJS one: as ESM is asynchronous and CJS synchronous, I think it's the only way to combine both of them (without renaming files or change `package.json`'s type).

But I'm really not sure and open to any suggestion! :hugs:

Reply:
Grrr, I hate this nonsense (not your fault lol).

Technically the code looks like the `createIconAssetsPlugin` could be undefined when the plugin is run but I guess this will never happen. I guess this is why I create a `.cjs` module for markdown.

Let's keep it like this :+1: